code/game/atoms_movable.dm

```diff
@@ -351,6 +351,18 @@
 		var/mob/buckled_mob = m
 		buckled_mob.set_glide_size(target)
 
+///meant for moves with zero side effects. only use for objects that arent supposed to interact with anything else (like camera mobs)
+/atom/movable/proc/abstract_move(atom/new_loc)
+	var/atom/old_loc = update_loc(new_loc)
+	Moved(old_loc)
+
+///meant to be used for all location changes. any instances of setting loc directly (for movables) should instead use this
```

Comment:
Can you make this documentation more clear on when you'd want this over forceMove?

Reply:
update_loc() or abstract_move()?

Reply:
update_loc() is used inside of all movement procs if thats what youre asking. it replaces all cases of loc = some_atoms

Reply:
Both, I guess.

As a developer, I might see "setting loc directly" and think "ooh, yeah I want that", when in reality I want forceMove.

Reply:
docs updated